packages/nuxt/src/composables/use-plugin.ts

```diff
@@ -8,7 +8,7 @@ import type { VuesticOptions } from './../types';
 /** Register vuestic nuxt plugin */
 export const useVuesticPlugin = (options: VuesticOptions) => {
   addPluginTemplate({
-    src: resolve(distDir, './runtime/plugin.mjs'),
+    src: resolve(distDir, './runtime/plugin.ts'),
```

Comment:
Must stay `.mjs`, cuz `ts` file is compiled to `mjs`.

Were there any issues with that before?

Reply:
Without this, the module is not compiled

Reply:
It's compiled and with `.ts` here will be a wrong path (dist module.mjs on the screen):

<img width="910" alt="image" src="https://user-images.githubusercontent.com/64714442/189872667-5d70fd23-6de4-46e0-9356-40f93bc84123.png">

Reply:
I didn't change the filename. 
With src dir (`resolve(distDir, "./runtime/plugin.mjs")`) we can't get types using `dev:prepare`. We get an error that there is no such file. With an extension .ts everything is fine

Reply:
And the `filename` option specifies how the file will be named after compilation - there should be `mjs`, yes